### PR TITLE
Collect bootkube logs if MCS failed to start

### PR DESCRIPTION
**Pull request by vrutkovs**
If MCS endpoint is not available even from bootstrap node the playbook would output bootkube service logs

### CODE REVIEW

**Comment by vrutkovs** [comment]
This works, but requires #10886 to pass operators status check task

File: playbooks/deploy_cluster_40.yml [hunk 1]
```diff
@@ -56,6 +56,37 @@
...
+    - mcs.status == 200
+    ignore_errors: true
+  - when: mcs is failed
+    block:
+    - name: Get node logs
+      command: journalctl --no-pager -u bootkube
+      register: bootkube_logs
+      ignore_errors: true
+    - name: Collect a list of containers
+      command: crictl ps -a -q
```
Comment (DanyC97) [inline playbooks/deploy_cluster_40.yml]: silly question - if you running using RHCOS image, then i suspect you need sudo and if so do you need it here ? or you have  that covered in ansible.cfg maybe ?

Comment (vrutkovs) [reply playbooks/deploy_cluster_40.yml]: This playbook won't work on RHCOS - a usual installer should be used instead.

The playbook assumes root account or sudo with `become` on target hosts

**Pull request merged**